### PR TITLE
Saad

### PR DESCRIPTION
**Pull request by raosaad373**

### CODE REVIEW

File: routes/solutionsController.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ router.get(
     // });
 
     // if (valid_user) {
-    SolutionModel.find({ user_role: req.params.role, status: "Pending" })
+    SolutionModel.find({ user_role: req.params.role, $or:[ {'status':'Pending'}, {'status':'Active'}, {'status':'Accepted'}, {'status':'InActive'} ] })
```
Comment (thealitahir) [inline routes/solutionsController.js:38]: I specifically asked you to use the "Not" query, you still used or, please change it

File: routes/solutionsController.js [hunk 2]
```diff
@@ -527,6 +527,95 @@ router.put("/solutionAccepted", async (req, res) => {
...
+        .populate("user")
+        .exec((err, data) => {
+          if (!err) {
+            resolve(data);
+          } else {
+            reject(err);
+          }
+        });
+    });
+    if (updated_solution) {
```
Comment (thealitahir) [inline routes/solutionsController.js]: what is this if condition doing?

File: routes/solutionsController.js (hunk 1 above)
Comment (raosaad373) [reply routes/solutionsController.js:38]: Not query not giving the desired result

File: routes/solutionsController.js (hunk 2 above)
Comment (raosaad373) [reply routes/solutionsController.js]: I just use your previous update code and update the status by getting it from body

**Review by thealitahir — APPROVED** [review]

**Pull request merged**